### DISCUSSION TITLE
Stuck on Docker Install

### DISCUSSION THREAD

**Discussion by makercurious** [meltaxa/jukebox-portal#18] [General]
Cool project. Setting up with mp3 direct play and am at USING DOCKER in instructions. When I run docker with this line after **curl -L https://raw.githubusercontent.com/meltaxa/musicfig/master/install.sh | bash -s -- --docker** and **lsusb | grep 0e6f:0241** which reports Bus 001 Device 005: ID 0e6f:0241 Logic 3,  **docker run -v /home/pi/musicfig/config.py :/config -p 5000:5000 --device=/dev/bus/usb/001/005 --device=/dev/snd meltaxa/musicfig** returns this: 

![Docker Help](https://user-images.githubusercontent.com/95706761/145076223-17561673-fd83-433d-a550-371770bd0f4b.png)

Am I missing something? Thank you for any help you can offer.

**Comment by makercurious** [comment]
Ok, I'm still looking for help. I got through the above with the following:

i@raspberrypi:~/musicfig $ sudo docker run -v /home/pi/musicfig:/config --device=/dev/bus/usb/001/005--device=/dev/snd meltaxa/musicfig
Unable to find image 'meltaxa/musicfig:latest' locally
latest: Pulling from meltaxa/musicfig
2703ee001e01: Pull complete
5f00d516e34b: Pull complete
ac789cd7e7d8: Pull complete
5e16bfec3cd4: Pull complete
6fd8364950cc: Pull complete
8eedb84a82da: Pull complete
211966866b96: Pull complete
e79d669a62ad: Pull complete
292bd2ef7b0a: Pull complete
324c4a31d52e: Pull complete
3ce9d6be7632: Pull complete
effce8ed703b: Pull complete
3bb5c6652bae: Pull complete
Digest: sha256:b95c89b3ff2732a95ad8a15d4318209fc1722acf1b1411b7466c1117588adf2b
Status: Downloaded newer image for meltaxa/musicfig:latest

I still couldn't get things working so I unplugged and replugged the LEGO stage in the USB slot of the Raspberry Pi and received the BUS and DEVICE, changed the command line and here's what I got:

pi@raspberrypi:~/musicfig $ sudo docker run -v /home/pi/musicfig:/config --device=/dev/bus/usb/001/010--device=/dev/snd meltaxa/musicfig

docker: Error response from daemon: error gathering device information while adding custom device "/dev/bus/usb/001/010--device=/dev/snd": no such file or directory.
ERRO[0001] error waiting for container: context canceled

Looking for any help you can provide on the last error. Thanks.

**Comment by meltaxa** [comment]
I've pushed a new version on Github. Pull the latest code or Docker image.
The 99-lego.rules did not copy during your initial install.
